ci/pipeline.yml

```
@@ -316,6 +321,127 @@ jobs:
         file: bosh-agent-ci/ci/tasks/test-unit-windows.yml
         tags: [*worker_tag_windows]
 
+  - name: test-integration-resolute
+    serial: true
+    plan:
+      - in_parallel:
+          - get: bosh-agent
+            trigger: true
+          - get: bosh-agent-ci
+          - get: bosh-ecosystem-concourse
+            tags: [ broadcom ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)